applicationconfig/client/src/test/java/com/azure/applicationconfig/ConfigurationClientTest.java

```diff
@@ -55,38 +54,36 @@
     public TestName testName = new TestName();
 
     @Before
-    public void beforeTest() throws Exception {
+    public void beforeTest() throws IOException, NoSuchAlgorithmException, InvalidKeyException {
         final TestMode testMode = getTestMode();
-        final String playbackUri = getPlaybackUri(testMode);
-        final HttpPipelinePolicy loggingPolicy = new HttpLoggingPolicy(HttpLogDetailLevel.BODY_AND_HEADERS);
-        final String connectionString;
-        final HttpPipeline pipeline;
 
         interceptorManager = InterceptorManager.create(testName.getMethodName(), testMode);
 
         if (interceptorManager.isPlaybackMode()) {
             logger.info("PLAYBACK MODE");
 
-            connectionString = "endpoint=" + playbackUri + ";Id=0000000000000;Secret=MDAwMDAw";
+            final String playbackUri = getPlaybackUri(testMode);
+            final String connectionString = "endpoint=" + playbackUri + ";Id=0000000000000;Secret=MDAwMDAw";
 
-            List<HttpPipelinePolicy> policies = ConfigurationClient.getDefaultPolicies(connectionString);
-            policies.add(loggingPolicy);
-
-            pipeline = new HttpPipeline(interceptorManager.getPlaybackClient(), policies);
+            client = ConfigurationClient.builder()
+                    .credentials(new ConfigurationClientCredentials(connectionString))
+                    .httpClient(interceptorManager.getPlaybackClient())
+                    .httpLogDetailLevel(HttpLogDetailLevel.BODY_AND_HEADERS)
+                    .build();
         } else {
             logger.info("RECORD MODE");
 
-            connectionString = System.getenv("AZCONFIG_CONNECTION_STRING");
+            final String connectionString = System.getenv("AZCONFIG_CONNECTION_STRING");
             Objects.requireNonNull(connectionString, "AZCONFIG_CONNECTION_STRING expected to be set.");
 
-            List<HttpPipelinePolicy> policies = ConfigurationClient.getDefaultPolicies(connectionString);
-            policies.add(interceptorManager.getRecordPolicy());
-            policies.add(loggingPolicy);
-
-            pipeline = new HttpPipeline(HttpClient.createDefault().wiretap(true), policies);
+            client = ConfigurationClient.builder()
+                    .credentials(new ConfigurationClientCredentials(connectionString))
+                    .httpClient(HttpClient.createDefault().wiretap(true))
+                    .httpLogDetailLevel(HttpLogDetailLevel.BODY_AND_HEADERS)
+                    .addPolicy(interceptorManager.getRecordPolicy())
```

Comment:
@JonathanGiles @conniey just double checking so we use `add**` prefix (e.g. addPolicy) as the general pattern to add items to collection configuration items?

Reply:
I think that is best - thanks for double checking.